distribution/docker/DockerfileBuildTarAdvanced

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
nit: this looks mostly a copy of other Dockerfile , it would be nice to reuse same if possible.

Reply:
Also. There are some changes between `DockerfileBuildTarAdvanced` and original `Dockerfile`, such as :
1. Prepare test data.
2. mvn package out of `Dockerfile`.
3. Mkdir necessary for IT.

So that we can't use the original Dockerfile here.